altair/examples/dendrogram.py

```diff
@@ -95,22 +100,25 @@ def get_df_coord(den):
 base = alt.Chart(source)
 
 # the U shape is composed of a shoulder plus two arms
-shoulder_encoding = [
-        alt.X("xk2:Q", title=""),
-        alt.X2("xk3:Q"),
-        alt.Y("yk2:Q", title="")
-]
-arm1_encoding = [alt.X("xk1:Q"), alt.Y("yk1:Q"), alt.Y2("yk2:Q")]
-arm2_encoding = [alt.X("xk3:Q"), alt.Y("yk3:Q"), alt.Y2("yk4:Q")]
-
-shoulder = base.encode(*shoulder_encoding).mark_rule()
-arm1 = base.encode(*arm1_encoding).mark_rule()
-arm2 = base.encode(*arm2_encoding).mark_rule()
+shoulder = base.encode(
+    alt.X("xk2:Q", title=""),
+    alt.X2("xk3:Q"),
+    alt.Y("yk2:Q", title="")
+).mark_rule()
```

Comment:
```suggestion
shoulder = base.mark_rule().encode(
    alt.X("xk2:Q", title=""),
    alt.X2("xk3:Q"),
    alt.Y("yk2:Q", title="")
)
```

Most of the documentation uses this so let's do that just to keep it consistent even if both work.